#!/usr/bin/python3
def raise_exception_msg(message=""):
  raise NameError("Sorry, The name Is Incorrect.. Try Again Please")
